app/retail/templates/index.html

```diff
@@ -417,6 +417,35 @@ <h4 class="">{% trans "Press" %}</h4>
           </div>
         </div>
       </div>
+      <div class="container-fluid pt-0" style="background-color: #eee;">
```

Comment:
the `background-color` seems to be overridden [here](https://d3vv6lp55qjaqc.cloudfront.net/items/1f0v0R0k1A3T1q471q0Q/Screen%20Shot%202018-05-07%20at%209.15.34%20AM.png)